### DISCUSSION TITLE
camtasia, snagit & keepass versions outdated

### DISCUSSION THREAD

**Discussion by umol-unibas** [pauby/ChocoPackages#211] [General]
We have noticed some repos not being up to date, e. g. camtasia, snagit & keepass, and don't match the chcolatey.org page. Do you plan to update these? Would be great!

**Comment by pauby** [comment]
> We have noticed some repos not being up to date,

I'm unsure what you mean? Are the packages not up to date or do you mean the source code in this repository doesn't match what is submitted as the packages (i.e. the version number in the nuspec file for example)?

If it's the code, see [here](https://github.com/pauby/ChocoPackages/blob/master/CONTRIBUTING.md#updating-packages-in-the-automatic-folder). The 'code' is updated when built and isn't pushed back here. That's intentional.

**Reply by umol-unibas** [reply]
Okay! We referred to the code-base. Thanks for your explanations - discussion can be closed
